internal/command/deploy/deploy_first.go

```diff
@@ -14,14 +16,28 @@ func (md *machineDeployment) provisionFirstDeploy(ctx context.Context, allocPubl
 		return nil
 	}
 	if err := md.provisionIpsOnFirstDeploy(ctx, allocPublicIPs); err != nil {
-		fmt.Fprintf(md.io.ErrOut, "Failed to provision IP addresses, use `fly ips` commands to remmediate it. ERROR: %s", err)
+		fmt.Fprintf(md.io.ErrOut, "Failed to provision IP addresses. Use `fly ips` commands to remediate it. ERROR: %s", err)
 	}
 	if err := md.provisionVolumesOnFirstDeploy(ctx); err != nil {
 		return fmt.Errorf("failed to provision seed volumes: %w", err)
 	}
+
+	// Provision Sentry on first deployment, unless we're using CI, where it's more likely to see ephemeral applications
+	if !env.IsCI() && md.provisionExtensions {
```

Comment:
what if it sets the default value of `--provision-extensions` to `!env.IsCI()` instead of this? it will behave as we want and still allow testing on CI if needed.